src/rpc/server.cpp

```diff
@@ -280,9 +281,9 @@ static const CRPCCommand vRPCCommands[] =
 { //  category              name                      actor (function)         okSafe argNames
   //  --------------------- ------------------------  -----------------------  ------ ----------
     /* Overall control/query calls */
-    { "control",            "help",                   &help,                   true,  {"command"}  },
-    { "control",            "stop",                   &stop,                   true,  {}  },
-    { "control",            "uptime",                 &uptime,                 true,  {}  },
+    { "control",            "/v1/*"    , "help",                   &help,                   true,  {"command"}  },
```

Comment:
it's a bit strange that the `help` RPC returns help for non-wallet commands when called on a wallet endpoint